### DISCUSSION TITLE
Calendar: Apply mask on Calendar

### DISCUSSION THREAD

**Discussion by 1Map** [primefaces/community#833] [PrimeVue]
Is it possible that Calendar component have a similar property as in InputMask (mask), for example:

`<Calendar :mask="9999-99-99" />`

**Comment by tugcekucukoglu** [comment]
We can wait for more demand.

**Comment by alqa-dash** [comment]
Nice to see that I'm not the only one here. I terribly need mask feature in Calendar component. 

My case for now is as follows: manual time input by format hh24:mm with 15-minutes step.
